src/main/java/ch/uzh/ifi/hase/soprafs24/controller/GameController.java

```diff
@@ -1,22 +1,25 @@
 package ch.uzh.ifi.hase.soprafs24.controller;
 
+import ch.uzh.ifi.hase.soprafs24.model.game.Game;
 import ch.uzh.ifi.hase.soprafs24.model.game.GameParameters;
+import ch.uzh.ifi.hase.soprafs24.rest.dto.PostGameStartDTO;
 import ch.uzh.ifi.hase.soprafs24.service.GameService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
-@RestController("/game")
+@RestController
+@RequestMapping("/game")
 @RequiredArgsConstructor
 public class GameController {
 
     private final GameService gameService;
 
-    @PostMapping
-    @ResponseBody
+    @PostMapping("")
     @ResponseStatus(HttpStatus.CREATED)
-    public GameParameters startGame(@RequestBody GameParameters gameParameters) {
+    public PostGameStartDTO createGame(@RequestBody GameParameters gameParameters) {
```

Comment:
Don't we need a header parameter sessionToken or similar additionally? (we have the same issue for GET auth/token and DELETE auth/token)?

Subsequently, I'd hand that sessionToken into createGame and then look up the userId based on that.